docs/installation/docker-compose_install.md

```diff
@@ -0,0 +1,174 @@
+Create the following sub-folders: `mongodb`, `minio`, and `digital-twins`.
+The file system structure should be like this:
+
+```bash
+└── odtp
+    ├── mongodb
+    ├── minio
+    └── digital-twins
+```
+
+Save the name of the path names for later use during the installation:
+
+- absolute path to the `digital-twins` folder as `[ODTP_PATH]`
+- absolute path to the `minio` folder as `[MINIO_PATH]`
+- absolute path to the `mongodb` folder as `[MONGODB_PATH]`
+
+## 2. Get your your Github Token
+
+To complete the configuration of the `.env` we will need your Github Token: 
+
+Go to the [Github Token page](https://github.com/settings/tokens) and generate a new classic token with full access rights.
+Choose an appropriate expiration data to work with the token.
+
+Save the name of the `[GITHUB_TOKEN]` for later use during the installation.
+
+## 3. Clone the ODTP Repository
+
+Pull the [ODTP](https://github.com/odtp-org/odtp/tree/main) repository.
+We recommend to do it in the same folder you created before (e.g.. `odtp`)
+
+```
+git clone https://github.com/odtp-org/odtp.git
+```
+
+Afterwards your folder structure will look like this:
+
+```bash
+└── odtp
+    ├── mongodb
+    ├── minio
+    └── digital-twins
+    └── odtp
+        └── compose.yml
+        └── .env.dist.compose
+        ...
+```
+
+## 4. Setup the enviroment variables
+
+The `.env` should be completed by adding administrator users, passwords and configuration for the different services: 
+
+```bash
+cd odtp
+cp .env.dist.compose .env
+```
+
+Then fill in your crendentials into `.env` as follows:
+
+Decide on crendentials for the different services
+
+- user and password for the Mongodb: `[MONGO_DB_USER]`, `[MONGO_DB_PASSWORD]` 
+- user and password for Mongodb Express: `[MONGO_EXPRESS_USER]`, `[MONGO_EXPRESS_PASSWORD]` 
+- user and password for Minio: `[MINIO_ROOT_USER]`, `[MINIO_ROOT_PASSWORD]` 
+
+Decide on a port to run the OTPD Dasboard on: `[DASHBOARD_PORT]`, for example `8003`
+
+```yaml
+# environment variables for installation with docker compose
+# -----------------------------------------------------------
+# fill these variables in case you want to install odtp with
+# docker compose
+
+# local setup and compose
+
+# odtp db instance in the mongo db: "odtp"
+ODTP_MONGO_DB=odtp
+# s3 bucket name: "odtp" 
+ODTP_BUCKET_NAME=odtp
+
+# s3 access and secret key
+ODTP_ACCESS_KEY=[MINIO_ROOT_USER]      
+ODTP_SECRET_KEY=[MINIO_ROOT_PASSWORD]
+
+# your github token
+GITHUB_TOKEN=[GITHUB_TOKEN]
+
+# mongodb user and password
+MONGO_DB_USER=[MONGO_DB_USER]
+MONGO_DB_PASSWORD=[MONGO_DB_PASSWORD]
+
+# mongoexpress user and password
+MONGO_EXPRESS_USER=[MONGO_EXPRESS_USER]
+MONGO_EXPRESS_PASSWORD=[MONGO_EXPRESS_PASSWORD]
+
+# absolute path for docker volumes
+ODTP_PATH=[ODTP_PATH]
+MINIO_PATH=[ODTP_PATH]
+MONGODB_PATH=[ODTP_PATH]
+
+# Dashboard parameters
+ODTP_DASHBOARD_PORT=[DASHBOARD_PORT]
+ODTP_DASHBOARD_RELOAD=False 
```

Comment:
Shouldn't be `True` by defautl?

Reply:
No, True is for development. Maybe in the next version we should not expose this any more.